coding-standard-baseline/README.md

```diff
@@ -0,0 +1,35 @@
+# Magento 2 Coding Standard Action
+
+A Github Action that runs the Magento Coding Standard.
+
+## Inputs
+
+See the [action.yml](./action.yml)
+
+## Usage
+
+```yml
+name: Coding Standard baseline
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  coding-standard:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: mage-os/github-actions/coding-standard-baseline@main
+          php_version: 8.1 # Optional, will be used for Php version
+          composer_version: 2
+          version: 31 # Optional, will use the latest if omitted.
+          path: app/code # Optional, will be used when event is not a pull request.
```

Comment:
This is unused, isn't it?

Reply:
Hi @sprankhub  -  we have added our comments in README.md file,  if not required then we can remove that.

Reply:
If I am not mistaken, you never use a `path` parameter anywhere. Hence, it should be safe to remove it.